File: newcoder_pat_basic/1028/go/main.go
package main

import "fmt"

func main() {
	var sc [101]int
	var N, i, d int
	fmt.Scanf("%d", &N)
	for i = 0; i < N; i++ {
		fmt.Scanf("%d", &d)
		sc[d]++
	}
	fmt.Scanf("%d", &N)
	for i = 0; i < N-1; i++ {
		fmt.Scanf("%d", &d)
		fmt.Printf("%d ", sc[d])
	}
	fmt.Scanf("%d", &d)
	fmt.Printf("%d\n", sc[d])
}

File: newcoder_pat_basic/1028/c/main.c
#include <stdio.h>

int main() {
  int sc[101] = {0}, N, i, d;
  scanf("%d", &N);
  for(i = 0; i < N; ++i) {
    scanf("%d", &d);
    ++sc[d];
  }
  scanf("%d", &N);
  for(i = 0; i < N-1; ++i) {
    scanf("%d", &d);
    printf("%d ", sc[d]);
  }
  scanf("%d", &d);
  printf("%d\n", sc[d]);
  return 0;
}
